src/utils/validateIdentifierInputs.test.ts

```diff
@@ -44,7 +44,8 @@ describe("validateIdentifierInputs", () => {
           textInputType: "PAYMENT_RECEIPT",
         },
         {
-          label: "payment receipt with regex and empty value",
+          label: "payment receipt with regex and empty value optional",
+          isOptional: true,
```

Comment:
can we add a case for payment receipt where 
- isOptional is true
- regex is present
- value is not empty and fulfils regex

Reply:
it's all mixed up in the first unit test!